src/krux/krux_settings.py

```diff
@@ -215,6 +217,8 @@ def label(self, attr):
             "depth_per_pass": t("Depth Per Pass"),
             "part_size": t("Part Size"),
             "border_padding": t("Border Padding"),
+            "head_type": t("Tool head type"),
+            "head_power": t("Tool head speed / power"),
```

Comment:
Speed, in which head moves, is a different property, covered by "feed_rate". I would replace this whole sentence with "Power".